### DISCUSSION TITLE
Where is the PWA installed on Linux?

### DISCUSSION THREAD

**Discussion by Jason5Lee** [filips123/PWAsForFirefox#147] [Q&A] [answered]
Sorry for this stupid question I'm using it on Linux for the first time. On macOS or Windows it automatically creates a shortcut or application for me to start the PWA. But on Linux I cannot find the way to start it.

**Comment by filips123** [comment, marked as answer]
Which desktop environment do you use? If it supports XDG desktop entry specification (which most DEs support), it will be added as an entry on the start/application menu.

**Reply by Jason5Lee** [reply]
I'm not using a desktop environment. I open Firefox via X11.

**Reply by filips123** [reply]
Then you probably need to launch it using the command line. First run `firefoxpwa profile list` to view all profiles and web apps, then run `firefoxpwa site launch SITE-ID` to launch them.
